daemon/graphdriver/overlay/overlay.go

```diff
@@ -201,9 +200,15 @@ func parseOptions(options []string) (*overlayOptions, error) {
 }
 
 func supportsOverlay() error {
-	// We can try to modprobe overlay first before looking at
-	// proc/filesystems for when overlay is supported
-	exec.Command("modprobe", "overlay").Run()
+	// Access overlay filesystem so that Linux loads it (if possible).
+	mountTarget, err := ioutil.TempDir("", "supportsOverlay")
+	if err != nil {
+		logrus.WithField("storage-driver", "overlay2").Error("Could not create temporary directory, so assuming that 'overlay' is not supported.")
```

Comment:
The actual error need to be logged here as well, i.e. add `.WithError(err)`.

Reply:
Also, we have a convention to
 - start error messages with a lowercased letter
 - do not end error messages with `.`